From 54290356b5d44e45e1d6e5a54aee6eaf66a1e1d9 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Mon, 29 May 2023 15:32:24 +0300
Subject: [PATCH] lang/python/python-pyopenssl: bump to 23.1.0

---
 lang/python/python-pyopenssl/Makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/lang/python/python-pyopenssl/Makefile b/lang/python/python-pyopenssl/Makefile
index dc2a3de70..0885ab45e 100644
--- a/lang/python/python-pyopenssl/Makefile
+++ b/lang/python/python-pyopenssl/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump (hass)
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-pyopenssl
-PKG_VERSION:=21.0.0
-PKG_RELEASE:=1a
+PKG_VERSION:=23.1.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=pyOpenSSL
-PKG_HASH:=5e2d8c5e46d0d865ae933bef5230090bdaf5506281e9eec60fa250ee80600cb3
+PKG_HASH:=8cb78010a1eb2c8e24b851693b7b04dfe9b1dc0a5ab3843927b10a85b1dfbb2e
 
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
@@ -31,8 +32,7 @@ define Package/python3-pyopenssl
   URL:=https://pyopenssl.org/
   DEPENDS:= \
       +python3-light \
-      +python3-cryptography \
-      +python3-six
+      +python3-cryptography
 endef
 
 define Package/python3-pyopenssl/description
-- 
2.30.2

