From ea79157846288052c9f60f1fb33af148fc26cbea Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Fri, 9 Feb 2024 13:55:22 +0300
Subject: [PATCH] lang/python/python-boto3: bump to 1.33.13

---
 lang/python/python-boto3/Makefile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-boto3/Makefile b/lang/python/python-boto3/Makefile
index ebc38ecd7..918f3aec3 100644
--- a/lang/python/python-boto3/Makefile
+++ b/lang/python/python-boto3/Makefile
@@ -1,11 +1,13 @@
+# XXX bump (hass)
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-boto3
-PKG_VERSION:=1.28.7
+PKG_VERSION:=1.33.13
 PKG_RELEASE:=1
 
 PYPI_NAME:=boto3
-PKG_HASH:=ef6a465d3b25b89bcd00ff69675b33917166145e544735dcb9978091f5b0b752
+PKG_HASH:=0e966b8a475ecb06cc0846304454b8da2473d4c8198a45dfb2c5304871986883
 
 PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
 PKG_LICENSE:=Apache-2.0
-- 
2.30.2

