From 5e32220d4fd7e3ee900600e17a383e3caddd6196 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Sun, 11 Dec 2022 10:13:38 +0300
Subject: [PATCH] lang/python/python-awesomeversion: bump to 22.9.0

---
 lang/python/python-awesomeversion/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-awesomeversion/Makefile b/lang/python/python-awesomeversion/Makefile
index 73fc231eb..8815f8364 100644
--- a/lang/python/python-awesomeversion/Makefile
+++ b/lang/python/python-awesomeversion/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-awesomeversion
-PKG_VERSION:=21.8.1
+PKG_VERSION:=22.9.0
 PKG_RELEASE:=$(AUTORELEASE)
 
 PYPI_NAME:=awesomeversion
-PKG_HASH:=a72f3fff12df257e30f5e25a8550fc4da62e7591d5fdb70714f5c96827c37645
+PKG_HASH:=2f4190d333e81e10b2a4e156150ddb3596f5f11da67e9d51ba39057aa7a17f7e
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=MIT
-- 
2.30.2

