From 97bd0d39460710d4e0d57742c3480bb0c564b8cb Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Thu, 9 Mar 2023 19:06:56 +0300
Subject: [PATCH] lang/python/python-sqlalchemy: bump to 2.0.5.post1

---
 lang/python/python-sqlalchemy/Makefile | 14 +++++++++-----
 1 file changed, 9 insertions(+), 5 deletions(-)

diff --git a/lang/python/python-sqlalchemy/Makefile b/lang/python/python-sqlalchemy/Makefile
index f15cbb3ff..181550915 100644
--- a/lang/python/python-sqlalchemy/Makefile
+++ b/lang/python/python-sqlalchemy/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-sqlalchemy
-PKG_VERSION:=1.4.22
+PKG_VERSION:=2.0.5.post1
 PKG_RELEASE:=1
 
 PYPI_NAME:=SQLAlchemy
-PKG_HASH:=ec1be26cdccd60d180359a527d5980d959a26269a2c7b1b327a1eea0cab37ed8
+PKG_HASH:=13eb2a5882cfd9f4eedaaec14a5603a096f0125f7c3cb48611b3bfa3c253f25d
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=MIT
@@ -30,10 +31,13 @@ define Package/python3-sqlalchemy
   TITLE:=Database Abstraction Library
   URL:=https://www.sqlalchemy.org
   DEPENDS:= \
-	+python3-light \
-	+python3-urllib \
+	+python3-asyncio \
 	+python3-decimal \
-	+python3-logging
+	+python3-logging \
+	+python3-openssl \
+	+python3-sqlite3 \
+	+python3-unittest \
+	+python3-urllib
 endef
 
 define Package/python3-sqlalchemy/description
-- 
2.30.2

