From 8c3f4b268ca509accb924a3c9ef830b263b9e251 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Mon, 7 Aug 2023 14:29:02 +0300
Subject: [PATCH] lang/python/python-certifi: bump to 2023.7.22

---
 lang/python/python-certifi/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-certifi/Makefile b/lang/python/python-certifi/Makefile
index d5dd74519..ddeee4f9f 100644
--- a/lang/python/python-certifi/Makefile
+++ b/lang/python/python-certifi/Makefile
@@ -2,11 +2,12 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump (hass)
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-certifi
-PKG_VERSION:=2022.9.24
+PKG_VERSION:=2023.7.22
 PKG_RELEASE:=1
 
 PKG_MAINTAINER:=Eneas U de Queiroz <cotequeiroz@gmail.com>
@@ -14,7 +15,7 @@ PKG_LICENSE:=MPL-2.0
 PKG_LICENSE_FILES:=LICENSE
 
 PYPI_NAME:=certifi
-PKG_HASH:=0d9c601124e5a6ba9712dbc60d9c53c21e34f5f641fe83002317394311bdce14
+PKG_HASH:=539cc1d13202e33ca466e88b2807e29f4c13049d6d87031a3c110744495cb082
 
 include ../pypi.mk
 include $(INCLUDE_DIR)/package.mk
-- 
2.30.2

