From 8d2f143d94ca87c43846fb2e52039d135aca5d46 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Thu, 21 Dec 2023 16:39:36 +0300
Subject: [PATCH] lang/python/python-aiohttp: bump to 3.9.1

---
 lang/python/python-aiohttp/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-aiohttp/Makefile b/lang/python/python-aiohttp/Makefile
index e5246d2b4..873787ada 100644
--- a/lang/python/python-aiohttp/Makefile
+++ b/lang/python/python-aiohttp/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump (hass)
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-aiohttp
-PKG_VERSION:=3.8.5
+PKG_VERSION:=3.9.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=aiohttp
-PKG_HASH:=b9552ec52cc147dbf1944ac7ac98af7602e51ea2dcd076ed194ca3c0d1c7d0bc
+PKG_HASH:=8fc49a87ac269d4529da45871e2ffb6874e87779c3d0e2ccd813c0899221239d
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
-- 
2.30.2

