From 2740333837ef833ebd7242fc8223b8651db1ab92 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Thu, 7 Sep 2023 11:14:24 +0300
Subject: [PATCH] lang/python/python-bleak: bump to 0.21.0

---
 lang/python/python-bleak/Makefile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-bleak/Makefile b/lang/python/python-bleak/Makefile
index e27d4c86d..e20ad3aa5 100644
--- a/lang/python/python-bleak/Makefile
+++ b/lang/python/python-bleak/Makefile
@@ -4,14 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump (hass)
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-bleak
-PKG_VERSION:=0.20.2
+PKG_VERSION:=0.21.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=bleak
-PKG_HASH:=6c92a47abe34e6dea8ffc5cea9457cbff6e1be966854839dbc25cddb36b79ee4
+PKG_HASH:=227a88b78831e6e2322a9be209dce6628177382885e3065fbe9c25445a9bae79
 
 PKG_MAINTAINER:=Quintin Hill <stuff@quintin.me.uk>
 PKG_LICENSE:=MIT
-- 
2.30.2

