From 97f55d7048c8983831478dfedffde68b571e0f60 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Sun, 11 Dec 2022 21:08:50 +0300
Subject: [PATCH] lang/python/python-aiohttp: bump to 3.8.3

---
 lang/python/python-aiohttp/Makefile | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/lang/python/python-aiohttp/Makefile b/lang/python/python-aiohttp/Makefile
index 007f41b20..df616dbca 100644
--- a/lang/python/python-aiohttp/Makefile
+++ b/lang/python/python-aiohttp/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aiohttp
-PKG_VERSION:=3.7.4.post0
+PKG_VERSION:=3.8.3
 PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=493d3299ebe5f5a7c66b9819eacdcfbbaaf1a8e84911ddffcdc48888497afecf
+PKG_HASH:=3828fb41b7203176b82fe5d699e0d845435f2374750a44b480ea6b930f6be269
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
@@ -32,15 +33,16 @@ define Package/python3-aiohttp
   DEPENDS:= \
 	+python3-light \
 	+python3-attrs \
-	+python3-chardet \
+	+python3-charset-normalizer \
 	+python3-multidict \
 	+python3-async-timeout \
 	+python3-yarl \
+	+python3-frozenlist \
+	+python3-aiosignal \
 	+python3-logging \
 	+python3-codecs \
 	+python3-cgi \
-	+python3-openssl \
-	+python3-typing-extensions
+	+python3-openssl
 endef
 
 define Package/python3-aiohttp/description
-- 
2.30.2

