From 289739286213f39ef49d52d9c526a813c724744a Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Sat, 22 Jul 2023 13:56:57 +0300
Subject: [PATCH] lang/python/python-aiohttp: bump to 3.8.5

---
 lang/python/python-aiohttp/Makefile | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/lang/python/python-aiohttp/Makefile b/lang/python/python-aiohttp/Makefile
index 007f41b20..4378fb251 100644
--- a/lang/python/python-aiohttp/Makefile
+++ b/lang/python/python-aiohttp/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump (hass)
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aiohttp
-PKG_VERSION:=3.7.4.post0
+PKG_VERSION:=3.8.5
 PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=493d3299ebe5f5a7c66b9819eacdcfbbaaf1a8e84911ddffcdc48888497afecf
+PKG_HASH:=b9552ec52cc147dbf1944ac7ac98af7602e51ea2dcd076ed194ca3c0d1c7d0bc
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
@@ -32,15 +33,16 @@ define Package/python3-aiohttp
   DEPENDS:= \
 	+python3-light \
 	+python3-attrs \
-	+python3-chardet \
+	+python3-charset-normalizer \
 	+python3-multidict \
 	+python3-async-timeout \
 	+python3-yarl \
+	+python3-frozenlist \
+	+python3-aiosignal \
 	+python3-logging \
 	+python3-codecs \
 	+python3-cgi \
-	+python3-openssl \
-	+python3-typing-extensions
+	+python3-openssl
 endef
 
 define Package/python3-aiohttp/description
-- 
2.30.2

