From 4037523ce7869a61259d0bc3b340f4e58d770a7a Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Sat, 4 Feb 2023 12:15:22 +0300
Subject: [PATCH] lang/python/python-attrs: bump to 22.2.0

---
 lang/python/python-attrs/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-attrs/Makefile b/lang/python/python-attrs/Makefile
index f21a4ec37..390bc1e0d 100644
--- a/lang/python/python-attrs/Makefile
+++ b/lang/python/python-attrs/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-attrs
-PKG_VERSION:=21.4.0
+PKG_VERSION:=22.2.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=attrs
-PKG_HASH:=626ba8234211db98e869df76230a137c4c40a12d72445c45d5f5b716f076e2fd
+PKG_HASH:=c9227bfc2f01993c03f68db37d1d15c9690188323c067c641f1a35ca58185f99
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE
-- 
2.30.2

