From 27bca9b5d2849a1b2c1ba5df35261048b82c983d Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Sun, 11 Dec 2022 21:09:01 +0300
Subject: [PATCH] lang/python/python-psutil: bump to 5.9.4

---
 lang/python/python-psutil/Makefile | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/lang/python/python-psutil/Makefile b/lang/python/python-psutil/Makefile
index 59b4f0fd1..e56772bf4 100644
--- a/lang/python/python-psutil/Makefile
+++ b/lang/python/python-psutil/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-psutil
-PKG_VERSION:=5.9.0
+PKG_VERSION:=5.9.4
 PKG_RELEASE:=1
 
 PYPI_NAME:=psutil
-PKG_HASH:=869842dbd66bb80c3217158e629d6fceaecc3a3166d3d1faee515b05dd26ca25
+PKG_HASH:=3d7f9739eb435d4b1338944abe23f49584bde5395f27487d2ee25ad9a8774a62
 
 PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec1@gmail.com>
 PKG_LICENSE:=BSD 3-Clause
@@ -30,7 +31,7 @@ define Package/python3-psutil
   CATEGORY:=Languages
   TITLE:=psutil (process and system utilities)
   URL:=https://github.com/giampaolo/psutil
-  DEPENDS:=+python3-light
+  DEPENDS:=+python3
 endef
 
 define Package/python3-psutil/description
-- 
2.30.2

