From 84615a129bf46ca16a401257935cb42814189306 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Sun, 11 Dec 2022 10:13:39 +0300
Subject: [PATCH] lang/python/python-voluptuous: bump to 0.13.1

---
 lang/python/python-voluptuous/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-voluptuous/Makefile b/lang/python/python-voluptuous/Makefile
index c5f04a2a5..5836c4d05 100644
--- a/lang/python/python-voluptuous/Makefile
+++ b/lang/python/python-voluptuous/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-voluptuous
-PKG_VERSION:=0.12.1
+PKG_VERSION:=0.13.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=voluptuous
-PKG_HASH:=663572419281ddfaf4b4197fd4942d181630120fb39b333e3adad70aeb56444b
+PKG_HASH:=e8d31c20601d6773cb14d4c0f42aee29c6821bbd1018039aac7ac5605b489723
 
 PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=COPYING
-- 
2.30.2

