From 623c03abc4da5d9ba2111c01a9edc259641c8de0 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Sun, 11 Dec 2022 10:13:38 +0300
Subject: [PATCH] lang/python/python-aiohttp: bump to 3.8.1

---
 lang/python/python-aiohttp/Makefile | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/lang/python/python-aiohttp/Makefile b/lang/python/python-aiohttp/Makefile
index 007f41b20..e0fcf07af 100644
--- a/lang/python/python-aiohttp/Makefile
+++ b/lang/python/python-aiohttp/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aiohttp
-PKG_VERSION:=3.7.4.post0
+PKG_VERSION:=3.8.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=493d3299ebe5f5a7c66b9819eacdcfbbaaf1a8e84911ddffcdc48888497afecf
+PKG_HASH:=fc5471e1a54de15ef71c1bc6ebe80d4dc681ea600e68bfd1cbce40427f0b7578
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
@@ -32,15 +33,16 @@ define Package/python3-aiohttp
   DEPENDS:= \
 	+python3-light \
 	+python3-attrs \
-	+python3-chardet \
+	+python3-charset-normalizer \
 	+python3-multidict \
 	+python3-async-timeout \
 	+python3-yarl \
+	+python3-frozenlist \
+	+python3-aiosignal \
 	+python3-logging \
 	+python3-codecs \
 	+python3-cgi \
-	+python3-openssl \
-	+python3-typing-extensions
+	+python3-openssl
 endef
 
 define Package/python3-aiohttp/description
-- 
2.30.2

