From 534112637b609b5ef0db68dc9be0a57e27ae8228 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Tue, 31 Oct 2023 16:37:17 +0300
Subject: [PATCH] lang/python/python-jsonschema: bump to 4.19.1

---
 lang/python/python-jsonschema/Makefile | 18 +++++++++++++-----
 1 file changed, 13 insertions(+), 5 deletions(-)

diff --git a/lang/python/python-jsonschema/Makefile b/lang/python/python-jsonschema/Makefile
index 4c6e44d46..e15c77d6b 100644
--- a/lang/python/python-jsonschema/Makefile
+++ b/lang/python/python-jsonschema/Makefile
@@ -1,11 +1,13 @@
+# XXX bump (hass)
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-jsonschema
-PKG_VERSION:=4.17.3
-PKG_RELEASE:=3
+PKG_VERSION:=4.19.1
+PKG_RELEASE:=1
 
 PYPI_NAME:=jsonschema
-PKG_HASH:=0f864437ab8b6076ba6707453ef8f98a6a0d512a80e93f8abdb676f737ecb60d
+PKG_HASH:=ec84cc37cfa703ef7cd4928db24f9cb31428a5d0fa77747b8b51a847458e0bbf
 
 PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
 PKG_LICENSE:=MIT
@@ -23,8 +25,14 @@ define Package/python3-jsonschema
   SUBMENU:=Python
   TITLE:=An implementation of JSON Schema validation
   URL:=https://github.com/Julian/jsonschema
-  DEPENDS:=+python3-light +python3-attrs +python3-urllib \
-	  +python3-six +python3-pyrsistent +python3-setuptools
+  DEPENDS:= \
+	+python3-attrs \
+	+python3-urllib \
+	+python3-uuid \
+	+python3-jsonschema-specifications \
+	+python3-referencing
+# (rust-lang pkgs)
+  EXTRA_DEPENDS:=python3-rpds-py
 endef
 
 define Package/python3-jsonschema/description
-- 
2.30.2

