From 4b65fd5491be4e224c195072f1cf7e0773bfb35a Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Tue, 5 Mar 2024 14:32:06 +0300
Subject: [PATCH] lang/python/python-sqlalchemy: bump to 2.0.27

---
 lang/python/python-sqlalchemy/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-sqlalchemy/Makefile b/lang/python/python-sqlalchemy/Makefile
index c69c37f24..8f31ed6aa 100644
--- a/lang/python/python-sqlalchemy/Makefile
+++ b/lang/python/python-sqlalchemy/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump (hass)
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-sqlalchemy
-PKG_VERSION:=2.0.23
+PKG_VERSION:=2.0.27
 PKG_RELEASE:=1
 
 PYPI_NAME:=SQLAlchemy
-PKG_HASH:=c1bda93cbbe4aa2aa0aa8655c5aeda505cd219ff3e8da91d1d329e143e4aff69
+PKG_HASH:=86a6ed69a71fe6b88bf9331594fa390a2adda4a49b5c06f98e47bf0d392534f8
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=MIT
-- 
2.30.2

