From 2af0408ee73ed6e81fe8e95434c8ad09ab3333b3 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Mon, 29 May 2023 15:34:56 +0300
Subject: [PATCH] lang/python/python-websockets: bump to 11.0.1

---
 lang/python/python-websockets/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-websockets/Makefile b/lang/python/python-websockets/Makefile
index 737ea39f4..e1ae2baf1 100644
--- a/lang/python/python-websockets/Makefile
+++ b/lang/python/python-websockets/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump (hass)
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-websockets
-PKG_VERSION:=10.3
+PKG_VERSION:=11.0.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=websockets
-PKG_HASH:=fc06cc8073c8e87072138ba1e431300e2d408f054b27047d047b549455066ff4
+PKG_HASH:=369410925b240b30ef1c1deadbd6331e9cd865ad0b8966bf31e276cc8e0da159
 
 PKG_MAINTAINER:=Michal Vasilek <michal.vasilek@nic.cz>
 PKG_LICENSE:=BSD-3-Clause
-- 
2.30.2

