From c7a33caf8f1695cbf75bd374a565ec7549c11de2 Mon Sep 17 00:00:00 2001
From: The-BB <tun.chen.bo@gmail.com>
Date: Thu, 8 Feb 2024 10:21:34 +0300
Subject: [PATCH] libs/libgpiod: disable python3-gpiod

---
 libs/libgpiod/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/libgpiod/Makefile b/libs/libgpiod/Makefile
index ef15288b1..dd58fffee 100644
--- a/libs/libgpiod/Makefile
+++ b/libs/libgpiod/Makefile
@@ -107,4 +107,4 @@ endef
 
 $(eval $(call BuildPackage,libgpiod))
 $(eval $(call BuildPackage,gpiod-tools))
-$(eval $(call BuildPackage,python3-gpiod))
+#$(eval $(call BuildPackage,python3-gpiod))
-- 
2.30.2

