diff --git a/lang/python/Jinja2/Makefile b/lang/python/Jinja2/Makefile
index ba7d54259..09f47597b 100644
--- a/lang/python/Jinja2/Makefile
+++ b/lang/python/Jinja2/Makefile
@@ -1,15 +1,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=Jinja2
-PKG_VERSION:=3.0.3
+PKG_VERSION:=3.1.2
 PKG_RELEASE:=$(AUTORELEASE)
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=611bb273cd68f3b993fabdc4064fc858c5b47a973cb5aa7999ec1ba405c87cd7
+PKG_HASH:=31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852
 
 PKG_MAINTAINER:=Šimon Bořek <simon.borek@nic.cz>
 PKG_LICENSE:=BSD-3-Clause
diff --git a/lang/python/host-pip-requirements/Cython.txt b/lang/python/host-pip-requirements/Cython.txt
index be0f28a12..a3ac1502f 100644
--- a/lang/python/host-pip-requirements/Cython.txt
+++ b/lang/python/host-pip-requirements/Cython.txt
@@ -1 +1 @@
-Cython==0.29.26 --hash=sha256:af377d543a762867da11fcf6e558f7a4a535ff8693f30cce123fab10c00fa312
+Cython==0.29.30 --hash=sha256:2235b62da8fe6fa8b99422c8e583f2fb95e143867d337b5c75e4b9a1a865f9e3
diff --git a/lang/python/numpy/Makefile b/lang/python/numpy/Makefile
index b0860617c..067897948 100644
--- a/lang/python/numpy/Makefile
+++ b/lang/python/numpy/Makefile
@@ -2,15 +2,16 @@
 # Copyright (C) 2019 Jakub Piotr Cłapa <jpc@loee.pl>
 # Copyright (C) 2020 Alexandru Ardelean <ardeleanalex@gmail.com>
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=numpy
-PKG_VERSION:=1.21.5
+PKG_VERSION:=1.23.2
 PKG_RELEASE:=$(AUTORELEASE)
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=6a5928bc6241264dce5ed509e66f33676fc97f464e7a919edc672fb5532221ee
+PKG_HASH:=b78d00e48261fbbd04aa0d7427cf78d18401ee0abd89c7559bbf422e5b1c7d01
 
 PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
 
@@ -21,10 +22,11 @@ PKG_CPE_ID:=cpe:/a:numpy:numpy
 PKG_CONFIG_DEPENDS:= \
         CONFIG_NUMPY_OPENBLAS_SUPPORT
 
-# yes, zip... sigh
-PYPI_SOURCE_EXT:=zip
 HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=Cython
 
+# XXX currently supports only x86_64 AVX-512
+PYTHON3_PKG_SETUP_VARS:=NPY_DISABLE_SVML=1
+
 include ../pypi.mk
 include $(INCLUDE_DIR)/package.mk
 include ../python3-package.mk
@@ -35,34 +37,35 @@ define Package/python3-numpy
   CATEGORY:=Languages
   TITLE:=The fundamental package for scientific computing with Python
   URL:=https://www.numpy.org/
-  DEPENDS:=@!SOFT_FLOAT +INSTALL_GFORTRAN:libgfortran +python3 \
-	   +NUMPY_OPENBLAS_SUPPORT:openblas
+#  DEPENDS:=@!SOFT_FLOAT +INSTALL_GFORTRAN:libgfortran +python3 \
+#	   +NUMPY_OPENBLAS_SUPPORT:openblas
+  DEPENDS:=+python3
 endef
 
 # We need to duplicate the omitted arhitectures same as openblas
-define Package/python3-numpy/config
-menu "Configuration"
-depends on !powerpc
-depends on !arc
+#define Package/python3-numpy/config
+#menu "Configuration"
+#depends on !powerpc
+#depends on !arc
 
-config NUMPY_OPENBLAS_SUPPORT
-        bool "Enable OpenBLAS support"
-        default n
+#config NUMPY_OPENBLAS_SUPPORT
+#        bool "Enable OpenBLAS support"
+#        default n
 
-endmenu
-endef
+#endmenu
+#endef
 
-ifdef CONFIG_NUMPY_OPENBLAS_SUPPORT
+#ifdef CONFIG_NUMPY_OPENBLAS_SUPPORT
 define Build/Prepare/numpy-sitecfg
 	echo "[DEFAULT]"                                 >  $(PKG_BUILD_DIR)/site.cfg
-	echo "library_dirs = $(STAGING_DIR)/usr/lib"     >> $(PKG_BUILD_DIR)/site.cfg
-	echo "include_dirs = $(STAGING_DIR)/usr/include" >> $(PKG_BUILD_DIR)/site.cfg
+	echo "library_dirs = $(STAGING_DIR)/opt/lib"     >> $(PKG_BUILD_DIR)/site.cfg
+	echo "include_dirs = $(STAGING_DIR)/opt/include" >> $(PKG_BUILD_DIR)/site.cfg
 	echo                                             >> $(PKG_BUILD_DIR)/site.cfg
-	echo "[openblas]"                                >> $(PKG_BUILD_DIR)/site.cfg
-	echo "libraries = openblas"                      >> $(PKG_BUILD_DIR)/site.cfg
-	echo "runtime_library_dirs = /usr/lib"           >> $(PKG_BUILD_DIR)/site.cfg
+#	echo "[openblas]"                                >> $(PKG_BUILD_DIR)/site.cfg
+#	echo "libraries = openblas"                      >> $(PKG_BUILD_DIR)/site.cfg
+#	echo "runtime_library_dirs = /opt/lib"           >> $(PKG_BUILD_DIR)/site.cfg
 endef
-endif
+#endif
 
 define Build/Prepare
 	$(call Build/Prepare/Default)
@@ -77,6 +80,14 @@ For some modules to be available, the INSTALL_GFORTRAN symbol needs
 to be enabled in the OpenWrt core/toolchain.
 endef
 
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/opt/include/numpy
+	$(CP) $(PKG_BUILD_DIR)/numpy/core/include/numpy/* $(1)/opt/include/numpy
+	$(CP) \
+		$(PKG_BUILD_DIR)/build/src.linux-$(ARCH)-$(PYTHON3_VERSION)/numpy/core/include/numpy/*.h \
+			$(1)/opt/include/numpy
+endef
+
 $(eval $(call Py3Package,python3-numpy))
 $(eval $(call BuildPackage,python3-numpy))
 $(eval $(call BuildPackage,python3-numpy-src))
diff --git a/lang/python/python-aiohttp/Makefile b/lang/python/python-aiohttp/Makefile
index 007f41b20..e0fcf07af 100644
--- a/lang/python/python-aiohttp/Makefile
+++ b/lang/python/python-aiohttp/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aiohttp
-PKG_VERSION:=3.7.4.post0
+PKG_VERSION:=3.8.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=493d3299ebe5f5a7c66b9819eacdcfbbaaf1a8e84911ddffcdc48888497afecf
+PKG_HASH:=fc5471e1a54de15ef71c1bc6ebe80d4dc681ea600e68bfd1cbce40427f0b7578
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
@@ -32,15 +33,16 @@ define Package/python3-aiohttp
   DEPENDS:= \
 	+python3-light \
 	+python3-attrs \
-	+python3-chardet \
+	+python3-charset-normalizer \
 	+python3-multidict \
 	+python3-async-timeout \
 	+python3-yarl \
+	+python3-frozenlist \
+	+python3-aiosignal \
 	+python3-logging \
 	+python3-codecs \
 	+python3-cgi \
-	+python3-openssl \
-	+python3-typing-extensions
+	+python3-openssl
 endef
 
 define Package/python3-aiohttp/description
diff --git a/lang/python/python-async-timeout/Makefile b/lang/python/python-async-timeout/Makefile
index cb66dd01f..d2c64e544 100644
--- a/lang/python/python-async-timeout/Makefile
+++ b/lang/python/python-async-timeout/Makefile
@@ -4,16 +4,17 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=async-timeout
-PKG_VERSION:=3.0.1
-PKG_RELEASE:=2
+PKG_VERSION:=4.0.2
+PKG_RELEASE:=1
 
 PYPI_NAME:=async_timeout
 PYPI_SOURCE_NAME:=async-timeout
-PKG_HASH:=0c3c816a028d47f659d6ff5c745cb2acf1f966da1fe5c19c77a70282b25f4c5f
+PKG_HASH:=2163e1640ddb52b7a8c80d0a67a08587e5d245cc9c553a74a847056bc2976b15
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python-awesomeversion/Makefile b/lang/python/python-awesomeversion/Makefile
index 73fc231eb..8815f8364 100644
--- a/lang/python/python-awesomeversion/Makefile
+++ b/lang/python/python-awesomeversion/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-awesomeversion
-PKG_VERSION:=21.8.1
+PKG_VERSION:=22.9.0
 PKG_RELEASE:=$(AUTORELEASE)
 
 PYPI_NAME:=awesomeversion
-PKG_HASH:=a72f3fff12df257e30f5e25a8550fc4da62e7591d5fdb70714f5c96827c37645
+PKG_HASH:=2f4190d333e81e10b2a4e156150ddb3596f5f11da67e9d51ba39057aa7a17f7e
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-boto3/Makefile b/lang/python/python-boto3/Makefile
index 27f69b3c9..ae4c3ece6 100644
--- a/lang/python/python-boto3/Makefile
+++ b/lang/python/python-boto3/Makefile
@@ -1,11 +1,13 @@
+# XXX bump
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=boto3
-PKG_VERSION:=1.9.135
-PKG_RELEASE:=2
+PKG_VERSION:=1.16.52
+PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=5e5805992e873e5687b5ef3b4c56c386ccb4df1c3364f8b8601d289e2f275be1
+PKG_HASH:=360a9f805b11f2e468d48815193c55278765fb30b64350893ab63236a5034726
 
 PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python-botocore/Makefile b/lang/python/python-botocore/Makefile
index 739be1126..941669026 100644
--- a/lang/python/python-botocore/Makefile
+++ b/lang/python/python-botocore/Makefile
@@ -1,11 +1,13 @@
+# XXX bump
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=botocore
-PKG_VERSION:=1.12.66
-PKG_RELEASE:=2
+PKG_VERSION:=1.19.52
+PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=25c39ecc71356287cf79d66981ec77deca374e28043b19b2f818d48aa34272a1
+PKG_HASH:=dc5ec23deadbe9327d3c81d03fddf80805c549059baabd80dea605941fe6a221
 
 PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-ciso8601/Makefile b/lang/python/python-ciso8601/Makefile
index 02a82cf67..a2592c0cb 100644
--- a/lang/python/python-ciso8601/Makefile
+++ b/lang/python/python-ciso8601/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-ciso8601
-PKG_VERSION:=2.1.3
+PKG_VERSION:=2.2.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=ciso8601
-PKG_HASH:=bdbb5b366058b1c87735603b23060962c439ac9be66f1ae91e8c7dbd7d59e262
+PKG_HASH:=14ad817ed31a698372d42afa81b0173d71cd1d0b48b7499a2da2a01dcc8695e6
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-multidict/Makefile b/lang/python/python-multidict/Makefile
index b757b2cda..d08f55651 100644
--- a/lang/python/python-multidict/Makefile
+++ b/lang/python/python-multidict/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=multidict
-PKG_VERSION:=4.5.2
-PKG_RELEASE:=2
+PKG_VERSION:=6.0.2
+PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=024b8129695a952ebd93373e45b5d341dbb87c17ce49637b34000093f243dd4f
+PKG_HASH:=5ff3bd75f38e4c43f1f470f2df7a4d430b821c4ce22be384e1459cb57d6bb013
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python-netdisco/Makefile b/lang/python/python-netdisco/Makefile
index 377afead9..5efe5cc2d 100644
--- a/lang/python/python-netdisco/Makefile
+++ b/lang/python/python-netdisco/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-netdisco
-PKG_VERSION:=2.6.0
-PKG_RELEASE:=2
+PKG_VERSION:=3.0.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=netdisco
-PKG_HASH:=2b3aca14a1807712a053f11fd80dc251dd821ee4899aefece515287981817762
+PKG_HASH:=4dbb590482f377ccc461e01742707ccbe8f1d2d1b28578af91dc9b7febcdcbd2
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python-psutil/Makefile b/lang/python/python-psutil/Makefile
index 59b4f0fd1..e7a22d5c0 100644
--- a/lang/python/python-psutil/Makefile
+++ b/lang/python/python-psutil/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-psutil
-PKG_VERSION:=5.9.0
+PKG_VERSION:=5.9.2
 PKG_RELEASE:=1
 
 PYPI_NAME:=psutil
-PKG_HASH:=869842dbd66bb80c3217158e629d6fceaecc3a3166d3d1faee515b05dd26ca25
+PKG_HASH:=feb861a10b6c3bb00701063b37e4afc754f8217f0f09c42280586bd6ac712b5c
 
 PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec1@gmail.com>
 PKG_LICENSE:=BSD 3-Clause
@@ -30,7 +31,7 @@ define Package/python3-psutil
   CATEGORY:=Languages
   TITLE:=psutil (process and system utilities)
   URL:=https://github.com/giampaolo/psutil
-  DEPENDS:=+python3-light
+  DEPENDS:=+python3
 endef
 
 define Package/python3-psutil/description
diff --git a/lang/python/python-pyotp/Makefile b/lang/python/python-pyotp/Makefile
index 9cda8ac67..b847d4217 100644
--- a/lang/python/python-pyotp/Makefile
+++ b/lang/python/python-pyotp/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=pyotp
-PKG_VERSION:=2.3.0
-PKG_RELEASE:=2
+PKG_VERSION:=2.7.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=fc537e8acd985c5cbf51e11b7d53c42276fee017a73aec7c07380695671ca1a1
+PKG_HASH:=ce989faba0df77dc032b45e51c6cca42bcf20896c8d3d1e7cd759a53dc7d6cb5
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-pyserial/Makefile b/lang/python/python-pyserial/Makefile
index e74cc880e..3423e35a3 100644
--- a/lang/python/python-pyserial/Makefile
+++ b/lang/python/python-pyserial/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-pyserial
-PKG_VERSION:=3.4
-PKG_RELEASE:=2
+PKG_VERSION:=3.5
+PKG_RELEASE:=1
 
 PYPI_NAME:=pyserial
-PKG_HASH:=6e2d401fdee0eab996cf734e67773a0143b932772ca8b42451440cfed942c627
+PKG_HASH:=3c77e014170dfffbd816e6ffc205e9842efb10be9f58ec16d3e8675b4925cddb
 
 PKG_LICENSE:=BSD
 PKG_MAINTAINER:=Micke Prag <micke.prag@telldus.se>
diff --git a/lang/python/python-s3transfer/Makefile b/lang/python/python-s3transfer/Makefile
index a6e04f9f1..8ab5ce88c 100644
--- a/lang/python/python-s3transfer/Makefile
+++ b/lang/python/python-s3transfer/Makefile
@@ -1,11 +1,13 @@
+# XXX bump
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=s3transfer
-PKG_VERSION:=0.2.0
-PKG_RELEASE:=2
+PKG_VERSION:=0.3.3
+PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=f23d5cb7d862b104401d9021fc82e5fa0e0cf57b7660a1331425aab0c691d021
+PKG_HASH:=921a37e2aefc64145e7b73d50c71bb4f26f46e4c9f414dc648c6245ff92cf7db
 
 PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-sqlalchemy/Makefile b/lang/python/python-sqlalchemy/Makefile
index f15cbb3ff..f7d38ff32 100644
--- a/lang/python/python-sqlalchemy/Makefile
+++ b/lang/python/python-sqlalchemy/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-sqlalchemy
-PKG_VERSION:=1.4.22
+PKG_VERSION:=1.4.41
 PKG_RELEASE:=1
 
 PYPI_NAME:=SQLAlchemy
-PKG_HASH:=ec1be26cdccd60d180359a527d5980d959a26269a2c7b1b327a1eea0cab37ed8
+PKG_HASH:=0292f70d1797e3c54e862e6f30ae474014648bc9c723e14a2fda730adb0a9791
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=MIT
@@ -30,10 +31,17 @@ define Package/python3-sqlalchemy
   TITLE:=Database Abstraction Library
   URL:=https://www.sqlalchemy.org
   DEPENDS:= \
-	+python3-light \
-	+python3-urllib \
+	+python3-asyncio \
+	+python3-codecs \
 	+python3-decimal \
-	+python3-logging
+	+python3-greenlet \
+	+python3-logging \
+	+python3-ncurses \
+	+python3-openssl \
+	+python3-sqlite3 \
+	+python3-unittest \
+	+python3-urllib \
+	+python3-xml
 endef
 
 define Package/python3-sqlalchemy/description
diff --git a/lang/python/python-typing-extensions/Makefile b/lang/python/python-typing-extensions/Makefile
index b60c9ad10..8fcb1a220 100644
--- a/lang/python/python-typing-extensions/Makefile
+++ b/lang/python/python-typing-extensions/Makefile
@@ -4,16 +4,17 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-typing-extensions
-PKG_VERSION:=3.10.0.0
+PKG_VERSION:=4.4.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=typing-extensions
 PYPI_SOURCE_NAME:=typing_extensions
-PKG_HASH:=50b6f157849174217d0656f99dc82fe932884fb250826c18350e159ec6cdf342
+PKG_HASH:=1511434bb92bf8dd198c12b1cc812e800d4181cfcb867674e0f8279cc93087aa
 
 PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec1@gmail.com>
 PKG_LICENSE:=PSF-2.0
diff --git a/lang/python/python-typing-extensions/patches/600-fix-setup.patch b/lang/python/python-typing-extensions/patches/600-fix-setup.patch
new file mode 100644
index 000000000..345e01abc
--- /dev/null
+++ b/lang/python/python-typing-extensions/patches/600-fix-setup.patch
@@ -0,0 +1,53 @@
+--- /dev/null
++++ b/setup.cfg
+@@ -0,0 +1,7 @@
++[metadata]
++license-file = LICENSE
++
++[egg_info]
++tag_build = 
++tag_date = 0
++
+--- /dev/null
++++ b/setup.py
+@@ -0,0 +1,40 @@
++#!/usr/bin/env python
++# coding: utf-8
++
++from setuptools import setup
++
++version = '4.4.0'
++description = 'Backported and Experimental Type Hints for Python 3.7+'
++
++classifiers = [
++    'Development Status :: 5 - Production/Stable',
++    'Environment :: Console',
++    'Intended Audience :: Developers',
++    'License :: OSI Approved :: Python Software Foundation License',
++    'Operating System :: OS Independent',
++    'Programming Language :: Python :: 3',
++    'Programming Language :: Python :: 3 :: Only',
++    'Programming Language :: Python :: 3.6',
++    'Programming Language :: Python :: 3.7',
++    'Programming Language :: Python :: 3.8',
++    'Programming Language :: Python :: 3.9',
++    'Programming Language :: Python :: 3.10',
++    'Topic :: Software Development'
++]
++
++package_dir = 'src'
++
++setup(name='typing_extensions',
++      version=version,
++      description=description,
++      author='Guido van Rossum, Jukka Lehtosalo, Łukasz Langa, Michael Lee',
++      author_email='levkivskyi@gmail.com',
++      url='https://github.com/python/typing/blob/master/typing_extensions/README.rst',
++      license='PSF',
++      keywords='typing function annotations type hints hinting checking '
++               'checker typehints typehinting typechecking backport',
++      package_dir={'': package_dir},
++      py_modules=['typing_extensions'],
++      classifiers=classifiers,
++      install_requires=[])
++
diff --git a/lang/python/python-urllib3/Makefile b/lang/python/python-urllib3/Makefile
index 8812c0e43..054856a4d 100644
--- a/lang/python/python-urllib3/Makefile
+++ b/lang/python/python-urllib3/Makefile
@@ -4,11 +4,12 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-urllib3
-PKG_VERSION:=1.25.11
+PKG_VERSION:=1.26.5
 PKG_RELEASE:=1
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
@@ -17,7 +18,7 @@ PKG_LICENSE_FILES:=LICENSE.txt
 PKG_CPE_ID:=cpe:/a:urllib3_project:urllib3
 
 PYPI_NAME:=urllib3
-PKG_HASH:=8d7eaa5a82a1cac232164990f04874c594c9453ec55eef02eab885aa02fc17a2
+PKG_HASH:=a7acd0977125325f516bda9735fa7142b909a8d01e8b2e4c8108d0984e6e0098
 
 include ../pypi.mk
 include $(INCLUDE_DIR)/package.mk
diff --git a/lang/python/python-voluptuous/Makefile b/lang/python/python-voluptuous/Makefile
index c5f04a2a5..5836c4d05 100644
--- a/lang/python/python-voluptuous/Makefile
+++ b/lang/python/python-voluptuous/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-voluptuous
-PKG_VERSION:=0.12.1
+PKG_VERSION:=0.13.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=voluptuous
-PKG_HASH:=663572419281ddfaf4b4197fd4942d181630120fb39b333e3adad70aeb56444b
+PKG_HASH:=e8d31c20601d6773cb14d4c0f42aee29c6821bbd1018039aac7ac5605b489723
 
 PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=COPYING
diff --git a/lang/python/python-websockets/Makefile b/lang/python/python-websockets/Makefile
index b669fa77d..f66bb2d24 100644
--- a/lang/python/python-websockets/Makefile
+++ b/lang/python/python-websockets/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-websockets
-PKG_VERSION:=10.0
+PKG_VERSION:=10.2
 PKG_RELEASE:=$(AUTORELEASE)
 
 PYPI_NAME:=websockets
-PKG_HASH:=c4fc9a1d242317892590abe5b61a9127f1a61740477bfb121743f290b8054002
+PKG_HASH:=8351c3c86b08156337b0e4ece0e3c5ec3e01fcd14e8950996832a23c99416098
 
 PKG_MAINTAINER:=Michal Vasilek <michal.vasilek@nic.cz>
 PKG_LICENSE:=BSD-3-Clause
diff --git a/lang/python/python-yarl/Makefile b/lang/python/python-yarl/Makefile
index 95383c89d..1a2e0444d 100644
--- a/lang/python/python-yarl/Makefile
+++ b/lang/python/python-yarl/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=yarl
-PKG_VERSION:=1.6.3
+PKG_VERSION:=1.8.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=8a9066529240171b68893d60dca86a763eae2139dd42f42106b03cf4b426bf10
+PKG_HASH:=af887845b8c2e060eb5605ff72b6f2dd2aab7a761379373fd89d314f4752abbf
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
@@ -29,7 +30,7 @@ define Package/python3-yarl
   TITLE:=Yet another URL library
   URL:=https://github.com/aio-libs/yarl
   DEPENDS:= \
-  +python3-light \
+  +python3-codecs \
   +python3-multidict \
   +python3-urllib \
   +python3-idna
diff --git a/lang/python/python-zeroconf/Makefile b/lang/python/python-zeroconf/Makefile
index 9b3e220c0..0d1d3a93d 100644
--- a/lang/python/python-zeroconf/Makefile
+++ b/lang/python/python-zeroconf/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-zeroconf
-PKG_VERSION:=0.38.1
+PKG_VERSION:=0.39.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=zeroconf
-PKG_HASH:=10c501b25d8881b656e56c34674d98fe6bc752240a572e74f918bc849c93ba9c
+PKG_HASH:=b83cff68a0c8dcd2705b5e792796239accba2bfddb09bc8d05badc642f64e7f6
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python3-find-stdlib-depends.sh b/lang/python/python3-find-stdlib-depends.sh
index d0e983820..7724184bb 100644
--- a/lang/python/python3-find-stdlib-depends.sh
+++ b/lang/python/python3-find-stdlib-depends.sh
@@ -18,7 +18,6 @@ python3-dbm: dbm
 python3-decimal: decimal
 python3-distutils: distutils
 python3-email: email
-python3-gdbm: dbm.gnu
 python3-logging: logging
 python3-lzma: lzma
 python3-multiprocessing: multiprocessing
diff --git a/lang/python/python3-netifaces/Makefile b/lang/python/python3-netifaces/Makefile
index 02fc5c2b0..849a90e74 100644
--- a/lang/python/python3-netifaces/Makefile
+++ b/lang/python/python3-netifaces/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python3-netifaces
-PKG_VERSION:=0.10.9
-PKG_RELEASE:=2
+PKG_VERSION:=0.11.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=netifaces
-PKG_HASH:=2dee9ffdd16292878336a58d04a20f0ffe95555465fee7c9bd23b3490ef2abf3
+PKG_HASH:=043a79146eb2907edf439899f262b3dfe41717d34124298ed281139a8b93ca32
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python3-pyroute2/Makefile b/lang/python/python3-pyroute2/Makefile
index 1f8cf55e6..1174dd0c2 100644
--- a/lang/python/python3-pyroute2/Makefile
+++ b/lang/python/python3-pyroute2/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python3-pyroute2
-PKG_VERSION:=0.5.16
+PKG_VERSION:=0.6.13
 PKG_RELEASE:=1
 
 PYPI_NAME:=pyroute2
-PKG_HASH:=fe681a2d008cac815b9f287250d69a333fbfc2b2d89c37d58798104057149989
+PKG_HASH:=b03d49a581945fec2b1ec7d1d5125c6f40ba04ed11affc90c4caddc019e25792
 
 PKG_MAINTAINER:=Martin Matějek <martin.matejek@nic.cz>
 PKG_LICENSE:=GPL-2.0-or-later Apache-2.0
@@ -29,12 +30,13 @@ define Package/python3-pyroute2
   TITLE:=Python netlink library
   URL:=https://github.com/svinota/pyroute2
   DEPENDS:= \
-	  +python3-light \
-	  +python3-distutils \
-	  +python3-logging \
-	  +python3-multiprocessing \
-	  +python3-sqlite3 \
-	  +python3-ctypes
+	+python3-pyroute2-core \
+	+python3-pyroute2-ethtool \
+	+python3-pyroute2-ipdb \
+	+python3-pyroute2-ipset \
+	+python3-pyroute2-ndb \
+	+python3-pyroute2-nftables \
+	+python3-pyroute2-nslink
 endef
 
 define Package/python3-pyroute2/description
@@ -43,8 +45,6 @@ define Package/python3-pyroute2/description
   but now it supports many netlink protocols.
 endef
 
-PYTHON3_PKG_SETUP_ARGS:=
-
 $(eval $(call Py3Package,python3-pyroute2))
 $(eval $(call BuildPackage,python3-pyroute2))
 $(eval $(call BuildPackage,python3-pyroute2-src))
diff --git a/lang/python/xmltodict/Makefile b/lang/python/xmltodict/Makefile
index 44cf4cd50..c3e79fb90 100644
--- a/lang/python/xmltodict/Makefile
+++ b/lang/python/xmltodict/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-xmltodict
-PKG_VERSION:=0.12.0
-PKG_RELEASE:=2
+PKG_VERSION:=0.13.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=xmltodict
-PKG_HASH:=50d8c638ed7ecb88d90561beedbf720c9b4e851a9fa6c47ebd64e99d166d8a21
+PKG_HASH:=341595a488e3e01a85a9d8911d8912fd922ede5fecc4dce437eb4b6c8d037e56
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=MIT
diff --git a/libs/libjpeg-turbo/Makefile b/libs/libjpeg-turbo/Makefile
index 7cc012aa5..3f2c65d84 100644
--- a/libs/libjpeg-turbo/Makefile
+++ b/libs/libjpeg-turbo/Makefile
@@ -30,6 +30,13 @@ define Package/libjpeg-turbo
   PROVIDES:=libjpeg
 endef
 
+define Package/libturbojpeg
+  $(call Package/libjpeg-turbo/Default)
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE+= runtime library
+endef
+
 define Package/libjpeg-turbo-utils
   $(call Package/libjpeg-turbo/Default)
   SECTION:=utils
@@ -60,7 +67,7 @@ CMAKE_OPTIONS += \
 	-DWITH_JPEG8=OFF \
 	-DWITH_MEM_SRCDST=ON \
 	-DWITH_SIMD=O$(if $(findstring mips,$(CONFIG_ARCH)),FF,N) \
-	-DWITH_TURBOJPEG=OFF
+	-DWITH_TURBOJPEG=TRUE
 
 ifneq ($(findstring arm,$(CONFIG_ARCH)),)
 ifeq ($(findstring neon,$(CONFIG_CPU_TYPE)),)
@@ -73,6 +80,7 @@ define Build/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/opt/include/*.h $(1)/opt/include/
 	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libjpeg.so* $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libturbojpeg.so* $(1)/opt/lib/
 	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
 	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/*.pc $(1)/opt/lib/pkgconfig/
 	$(SED) 's,/opt/include,$$$${prefix}/include,g' $(1)/opt/lib/pkgconfig/libjpeg.pc
@@ -86,6 +94,11 @@ define Package/libjpeg-turbo/install
 	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libjpeg.so* $(1)/opt/lib
 endef
 
+define Package/libturbojpeg/install
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libturbojpeg.so* $(1)/opt/lib
+endef
+
 define Package/libjpeg-turbo-utils/install
 	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/rdjpgcom $(1)/opt/bin
@@ -96,4 +109,5 @@ define Package/libjpeg-turbo-utils/install
 endef
 
 $(eval $(call BuildPackage,libjpeg-turbo))
+$(eval $(call BuildPackage,libturbojpeg))
 $(eval $(call BuildPackage,libjpeg-turbo-utils))
diff --git a/libs/libre2/Makefile b/libs/libre2/Makefile
index 972cbf29b..d48ac935a 100644
--- a/libs/libre2/Makefile
+++ b/libs/libre2/Makefile
@@ -39,8 +39,8 @@ TARGET_LDFLAGS += \
 	-Wl,--as-needed,--gc-sections
 
 define Package/re2/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/libre2.so $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/lib/libre2.so $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,re2))
