diff --git a/lang/python/numpy/Makefile b/lang/python/numpy/Makefile
index bf71978..7f65771 100644
--- a/lang/python/numpy/Makefile
+++ b/lang/python/numpy/Makefile
@@ -35,34 +35,35 @@ define Package/python3-numpy
   CATEGORY:=Languages
   TITLE:=The fundamental package for scientific computing with Python
   URL:=https://www.numpy.org/
-  DEPENDS:=@!SOFT_FLOAT +INSTALL_GFORTRAN:libgfortran +python3 \
-	   +NUMPY_OPENBLAS_SUPPORT:openblas
+#  DEPENDS:=@!SOFT_FLOAT +INSTALL_GFORTRAN:libgfortran +python3 \
+#	   +NUMPY_OPENBLAS_SUPPORT:openblas
+  DEPENDS:=+python3
 endef
 
 # We need to duplicate the omitted arhitectures same as openblas
-define Package/python3-numpy/config
-menu "Configuration"
-depends on !powerpc
-depends on !arc
+#define Package/python3-numpy/config
+#menu "Configuration"
+#depends on !powerpc
+#depends on !arc
 
-config NUMPY_OPENBLAS_SUPPORT
-        bool "Enable OpenBLAS support"
-        default n
+#config NUMPY_OPENBLAS_SUPPORT
+#        bool "Enable OpenBLAS support"
+#        default n
 
-endmenu
-endef
+#endmenu
+#endef
 
-ifdef CONFIG_NUMPY_OPENBLAS_SUPPORT
+#ifdef CONFIG_NUMPY_OPENBLAS_SUPPORT
 define Build/Prepare/numpy-sitecfg
 	echo "[DEFAULT]"                                 >  $(PKG_BUILD_DIR)/site.cfg
-	echo "library_dirs = $(STAGING_DIR)/usr/lib"     >> $(PKG_BUILD_DIR)/site.cfg
-	echo "include_dirs = $(STAGING_DIR)/usr/include" >> $(PKG_BUILD_DIR)/site.cfg
+	echo "library_dirs = $(STAGING_DIR)/opt/lib"     >> $(PKG_BUILD_DIR)/site.cfg
+	echo "include_dirs = $(STAGING_DIR)/opt/include" >> $(PKG_BUILD_DIR)/site.cfg
 	echo                                             >> $(PKG_BUILD_DIR)/site.cfg
-	echo "[openblas]"                                >> $(PKG_BUILD_DIR)/site.cfg
-	echo "libraries = openblas"                      >> $(PKG_BUILD_DIR)/site.cfg
-	echo "runtime_library_dirs = /usr/lib"           >> $(PKG_BUILD_DIR)/site.cfg
+#	echo "[openblas]"                                >> $(PKG_BUILD_DIR)/site.cfg
+#	echo "libraries = openblas"                      >> $(PKG_BUILD_DIR)/site.cfg
+#	echo "runtime_library_dirs = /opt/lib"           >> $(PKG_BUILD_DIR)/site.cfg
 endef
-endif
+#endif
 
 define Build/Prepare
 	$(call Build/Prepare/Default)
diff --git a/lang/python/python-aiohttp/Makefile b/lang/python/python-aiohttp/Makefile
index 007f41b..e0fcf07 100644
--- a/lang/python/python-aiohttp/Makefile
+++ b/lang/python/python-aiohttp/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aiohttp
-PKG_VERSION:=3.7.4.post0
+PKG_VERSION:=3.8.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=493d3299ebe5f5a7c66b9819eacdcfbbaaf1a8e84911ddffcdc48888497afecf
+PKG_HASH:=fc5471e1a54de15ef71c1bc6ebe80d4dc681ea600e68bfd1cbce40427f0b7578
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
@@ -32,15 +33,16 @@ define Package/python3-aiohttp
   DEPENDS:= \
 	+python3-light \
 	+python3-attrs \
-	+python3-chardet \
+	+python3-charset-normalizer \
 	+python3-multidict \
 	+python3-async-timeout \
 	+python3-yarl \
+	+python3-frozenlist \
+	+python3-aiosignal \
 	+python3-logging \
 	+python3-codecs \
 	+python3-cgi \
-	+python3-openssl \
-	+python3-typing-extensions
+	+python3-openssl
 endef
 
 define Package/python3-aiohttp/description
diff --git a/lang/python/python-async-timeout/Makefile b/lang/python/python-async-timeout/Makefile
index cb66dd0..d485ea6 100644
--- a/lang/python/python-async-timeout/Makefile
+++ b/lang/python/python-async-timeout/Makefile
@@ -4,16 +4,17 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=async-timeout
-PKG_VERSION:=3.0.1
-PKG_RELEASE:=2
+PKG_VERSION:=4.0.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=async_timeout
 PYPI_SOURCE_NAME:=async-timeout
-PKG_HASH:=0c3c816a028d47f659d6ff5c745cb2acf1f966da1fe5c19c77a70282b25f4c5f
+PKG_HASH:=7d87a4e8adba8ededb52e579ce6bc8276985888913620c935094c2276fd83382
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
@@ -31,7 +32,8 @@ define Package/python3-async-timeout
   URL:=https://github.com/aio-libs/async-timeout
   DEPENDS:= \
   +python3-light \
-  +python3-asyncio
+  +python3-asyncio \
+  +python3-typing-extensions
 endef
 
 define Package/python3-async-timeout/description
diff --git a/lang/python/python-awesomeversion/Makefile b/lang/python/python-awesomeversion/Makefile
index 73fc231..2667a70 100644
--- a/lang/python/python-awesomeversion/Makefile
+++ b/lang/python/python-awesomeversion/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-awesomeversion
-PKG_VERSION:=21.8.1
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_VERSION:=21.11.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=awesomeversion
-PKG_HASH:=a72f3fff12df257e30f5e25a8550fc4da62e7591d5fdb70714f5c96827c37645
+PKG_HASH:=ef8f7bb458a6dd16ec95e28553df3b6c556de8bc93fb13f142eb95a357cfc3a0
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-boto3/Makefile b/lang/python/python-boto3/Makefile
index 27f69b3..646aa91 100644
--- a/lang/python/python-boto3/Makefile
+++ b/lang/python/python-boto3/Makefile
@@ -1,11 +1,13 @@
+# XXX bump
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=boto3
-PKG_VERSION:=1.9.135
-PKG_RELEASE:=2
+PKG_VERSION:=1.16.3
+PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=5e5805992e873e5687b5ef3b4c56c386ccb4df1c3364f8b8601d289e2f275be1
+PKG_HASH:=e2857738affb394bbe96473de2ed01331685d6e313bb1a3328fd5f47841429cc
 
 PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python-botocore/Makefile b/lang/python/python-botocore/Makefile
index 739be11..9014ae3 100644
--- a/lang/python/python-botocore/Makefile
+++ b/lang/python/python-botocore/Makefile
@@ -1,11 +1,13 @@
+# XXX bump
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=botocore
-PKG_VERSION:=1.12.66
-PKG_RELEASE:=2
+PKG_VERSION:=1.19.3
+PKG_RELEASE:=1a
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=25c39ecc71356287cf79d66981ec77deca374e28043b19b2f818d48aa34272a1
+PKG_HASH:=4ea4c74d244c1b4701387fd1abe6a5e1833dc621c6d39f8888f0bfa95ddd82f5
 
 PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-ciso8601/Makefile b/lang/python/python-ciso8601/Makefile
index 02a82cf..a2592c0 100644
--- a/lang/python/python-ciso8601/Makefile
+++ b/lang/python/python-ciso8601/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-ciso8601
-PKG_VERSION:=2.1.3
+PKG_VERSION:=2.2.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=ciso8601
-PKG_HASH:=bdbb5b366058b1c87735603b23060962c439ac9be66f1ae91e8c7dbd7d59e262
+PKG_HASH:=14ad817ed31a698372d42afa81b0173d71cd1d0b48b7499a2da2a01dcc8695e6
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-dns/Makefile b/lang/python/python-dns/Makefile
index 2c71f07..9f3c2aa 100644
--- a/lang/python/python-dns/Makefile
+++ b/lang/python/python-dns/Makefile
@@ -4,16 +4,17 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-dns
-PKG_VERSION:=1.16.0
-PKG_RELEASE:=2
+PKG_VERSION:=2.1.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=dnspython
 PYPI_SOURCE_EXT:=zip
-PKG_HASH:=36c5e8e38d4369a08b6780b7f27d790a292b2b08eea01607865bf0936c558e01
+PKG_HASH:=e4a87f0b573201a0f3727fa18a516b055fd1107e0e5477cded4a2de497df1dd4
 
 PKG_LICENSE:=ISC
 PKG_LICENSE_FILES:=LICENSE
diff --git a/lang/python/python-netdisco/Makefile b/lang/python/python-netdisco/Makefile
index 377afea..5efe5cc 100644
--- a/lang/python/python-netdisco/Makefile
+++ b/lang/python/python-netdisco/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-netdisco
-PKG_VERSION:=2.6.0
-PKG_RELEASE:=2
+PKG_VERSION:=3.0.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=netdisco
-PKG_HASH:=2b3aca14a1807712a053f11fd80dc251dd821ee4899aefece515287981817762
+PKG_HASH:=4dbb590482f377ccc461e01742707ccbe8f1d2d1b28578af91dc9b7febcdcbd2
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python-psutil/Makefile b/lang/python/python-psutil/Makefile
index cf3e2f8..09a050f 100644
--- a/lang/python/python-psutil/Makefile
+++ b/lang/python/python-psutil/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-psutil
 PKG_VERSION:=5.8.0
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PYPI_NAME:=psutil
 PKG_HASH:=0c9ccb99ab76025f2f0bbecf341d4656e9c1351db8cc8a03ccd62e318ab4b5c6
@@ -30,7 +30,7 @@ define Package/python3-psutil
   CATEGORY:=Languages
   TITLE:=psutil (process and system utilities)
   URL:=https://github.com/giampaolo/psutil
-  DEPENDS:=+python3-light
+  DEPENDS:=+python3
 endef
 
 define Package/python3-psutil/description
diff --git a/lang/python/python-pyotp/Makefile b/lang/python/python-pyotp/Makefile
index 9cda8ac..8b95ccf 100644
--- a/lang/python/python-pyotp/Makefile
+++ b/lang/python/python-pyotp/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=pyotp
-PKG_VERSION:=2.3.0
-PKG_RELEASE:=2
+PKG_VERSION:=2.6.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=fc537e8acd985c5cbf51e11b7d53c42276fee017a73aec7c07380695671ca1a1
+PKG_HASH:=d28ddfd40e0c1b6a6b9da961c7d47a10261fb58f378cb00f05ce88b26df9c432
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-pyserial/Makefile b/lang/python/python-pyserial/Makefile
index e74cc88..3423e35 100644
--- a/lang/python/python-pyserial/Makefile
+++ b/lang/python/python-pyserial/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-pyserial
-PKG_VERSION:=3.4
-PKG_RELEASE:=2
+PKG_VERSION:=3.5
+PKG_RELEASE:=1
 
 PYPI_NAME:=pyserial
-PKG_HASH:=6e2d401fdee0eab996cf734e67773a0143b932772ca8b42451440cfed942c627
+PKG_HASH:=3c77e014170dfffbd816e6ffc205e9842efb10be9f58ec16d3e8675b4925cddb
 
 PKG_LICENSE:=BSD
 PKG_MAINTAINER:=Micke Prag <micke.prag@telldus.se>
diff --git a/lang/python/python-requests/Makefile b/lang/python/python-requests/Makefile
index 5853897..97a72f5 100644
--- a/lang/python/python-requests/Makefile
+++ b/lang/python/python-requests/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-requests
 PKG_VERSION:=2.26.0
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>, Alexandru Ardelean <ardeleanalex@gmail.com>
 PKG_LICENSE:=Apache-2.0
@@ -34,7 +34,8 @@ define Package/python3-requests
 	  +python3-chardet  \
 	  +python3-idna  \
 	  +python3-urllib3  \
-	  +python3-certifi
+	  +python3-certifi \
+	  +python3-charset-normalizer
 endef
 
 define Package/python3-requests/description
diff --git a/lang/python/python-s3transfer/Makefile b/lang/python/python-s3transfer/Makefile
index a6e04f9..8ab5ce8 100644
--- a/lang/python/python-s3transfer/Makefile
+++ b/lang/python/python-s3transfer/Makefile
@@ -1,11 +1,13 @@
+# XXX bump
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=s3transfer
-PKG_VERSION:=0.2.0
-PKG_RELEASE:=2
+PKG_VERSION:=0.3.3
+PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=f23d5cb7d862b104401d9021fc82e5fa0e0cf57b7660a1331425aab0c691d021
+PKG_HASH:=921a37e2aefc64145e7b73d50c71bb4f26f46e4c9f414dc648c6245ff92cf7db
 
 PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
 PKG_LICENSE:=MIT
diff --git a/lang/python/python-sqlalchemy/Makefile b/lang/python/python-sqlalchemy/Makefile
index f15cbb3..bda4fe0 100644
--- a/lang/python/python-sqlalchemy/Makefile
+++ b/lang/python/python-sqlalchemy/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-sqlalchemy
-PKG_VERSION:=1.4.22
+PKG_VERSION:=1.4.27
 PKG_RELEASE:=1
 
 PYPI_NAME:=SQLAlchemy
-PKG_HASH:=ec1be26cdccd60d180359a527d5980d959a26269a2c7b1b327a1eea0cab37ed8
+PKG_HASH:=d768359daeb3a86644f3854c6659e4496a3e6bba2b4651ecc87ce7ad415b320c
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=MIT
@@ -33,7 +34,8 @@ define Package/python3-sqlalchemy
 	+python3-light \
 	+python3-urllib \
 	+python3-decimal \
-	+python3-logging
+	+python3-logging \
+	+python3-greenlet
 endef
 
 define Package/python3-sqlalchemy/description
diff --git a/lang/python/python-urllib3/Makefile b/lang/python/python-urllib3/Makefile
index 8812c0e..054856a 100644
--- a/lang/python/python-urllib3/Makefile
+++ b/lang/python/python-urllib3/Makefile
@@ -4,11 +4,12 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-urllib3
-PKG_VERSION:=1.25.11
+PKG_VERSION:=1.26.5
 PKG_RELEASE:=1
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
@@ -17,7 +18,7 @@ PKG_LICENSE_FILES:=LICENSE.txt
 PKG_CPE_ID:=cpe:/a:urllib3_project:urllib3
 
 PYPI_NAME:=urllib3
-PKG_HASH:=8d7eaa5a82a1cac232164990f04874c594c9453ec55eef02eab885aa02fc17a2
+PKG_HASH:=a7acd0977125325f516bda9735fa7142b909a8d01e8b2e4c8108d0984e6e0098
 
 include ../pypi.mk
 include $(INCLUDE_DIR)/package.mk
diff --git a/lang/python/python-voluptuous-serialize/Makefile b/lang/python/python-voluptuous-serialize/Makefile
index 738d8a0..a45c7e8 100644
--- a/lang/python/python-voluptuous-serialize/Makefile
+++ b/lang/python/python-voluptuous-serialize/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=voluptuous-serialize
-PKG_VERSION:=2.4.0
+PKG_VERSION:=2.5.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=c6ba17cb0301c18e8b955d89b85fa4aa05c05c80ab1e4873810900f757dceae4
+PKG_HASH:=5359f2e0a4f972ae03066e0777b4f0755c9226b2af099ca4fc55432efd1a447b
 
 PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python-voluptuous/Makefile b/lang/python/python-voluptuous/Makefile
index c5f04a2..e098a3e 100644
--- a/lang/python/python-voluptuous/Makefile
+++ b/lang/python/python-voluptuous/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-voluptuous
-PKG_VERSION:=0.12.1
+PKG_VERSION:=0.12.2
 PKG_RELEASE:=1
 
 PYPI_NAME:=voluptuous
-PKG_HASH:=663572419281ddfaf4b4197fd4942d181630120fb39b333e3adad70aeb56444b
+PKG_HASH:=4db1ac5079db9249820d49c891cb4660a6f8cae350491210abce741fabf56513
 
 PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=COPYING
diff --git a/lang/python/python-zeroconf/Makefile b/lang/python/python-zeroconf/Makefile
index 121a3f9..fdb52c6 100644
--- a/lang/python/python-zeroconf/Makefile
+++ b/lang/python/python-zeroconf/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-zeroconf
-PKG_VERSION:=0.29.0
+PKG_VERSION:=0.38.1
 PKG_RELEASE:=1
 
 PYPI_NAME:=zeroconf
-PKG_HASH:=7aefbb658b452b1fd7e51124364f938c6f5e42d6ea893fa2557bea8c06c540af
+PKG_HASH:=10c501b25d8881b656e56c34674d98fe6bc752240a572e74f918bc849c93ba9c
 
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/lang/python/python3-pyroute2/Makefile b/lang/python/python3-pyroute2/Makefile
index 1f8cf55..f3615b7 100644
--- a/lang/python/python3-pyroute2/Makefile
+++ b/lang/python/python3-pyroute2/Makefile
@@ -4,15 +4,16 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+# XXX bump
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python3-pyroute2
-PKG_VERSION:=0.5.16
+PKG_VERSION:=0.5.18
 PKG_RELEASE:=1
 
 PYPI_NAME:=pyroute2
-PKG_HASH:=fe681a2d008cac815b9f287250d69a333fbfc2b2d89c37d58798104057149989
+PKG_HASH:=08ac40cad52c0bb2e0f20087520599aa91fcce0b221dd26e9042330428810bc5
 
 PKG_MAINTAINER:=Martin Matějek <martin.matejek@nic.cz>
 PKG_LICENSE:=GPL-2.0-or-later Apache-2.0
@@ -34,7 +35,8 @@ define Package/python3-pyroute2
 	  +python3-logging \
 	  +python3-multiprocessing \
 	  +python3-sqlite3 \
-	  +python3-ctypes
+	  +python3-ctypes \
+	  +python3-psutil
 endef
 
 define Package/python3-pyroute2/description
diff --git a/net/scapy/Makefile b/net/scapy/Makefile
index 6cf711b..79b11c7 100644
--- a/net/scapy/Makefile
+++ b/net/scapy/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=scapy
 PKG_VERSION:=2.4.5
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_RELEASE:=1a
 
 PYPI_NAME:=$(PKG_NAME)
 PKG_HASH:=bc707e3604784496b6665a9e5b2a69c36cc9fb032af4864b29051531b24c8593
