--- a/turbojpeg.py
+++ b/turbojpeg.py
@@ -41,11 +41,7 @@ DEFAULT_LIB_PATHS = {
         '/opt/libjpeg-turbo/lib64/libturbojpeg.dylib',
         '/opt/homebrew/opt/jpeg-turbo/lib/libturbojpeg.dylib'
     ],
-    'Linux': [
-        '/usr/lib/x86_64-linux-gnu/libturbojpeg.so.0',
-        '/usr/lib64/libturbojpeg.so.0',
-        '/opt/libjpeg-turbo/lib64/libturbojpeg.so'
-    ],
+    'Linux': ['/opt/lib/libturbojpeg.so'],
     'FreeBSD': [
         '/usr/local/lib/libturbojpeg.so.0',
         '/usr/local/lib/libturbojpeg.so'
