--- a/turbojpeg.py
+++ b/turbojpeg.py
@@ -37,11 +37,7 @@ from struct import unpack, calcsize
 # default libTurboJPEG library path
 DEFAULT_LIB_PATHS = {
     'Darwin': ['/usr/local/opt/jpeg-turbo/lib/libturbojpeg.dylib'],
-    'Linux': [
-        '/usr/lib/x86_64-linux-gnu/libturbojpeg.so.0',
-        '/usr/lib64/libturbojpeg.so.0',
-        '/opt/libjpeg-turbo/lib64/libturbojpeg.so'
-    ],
+    'Linux': ['/opt/lib/libturbojpeg.so.0'],
     'FreeBSD': [
         '/usr/local/lib/libturbojpeg.so.0',
         '/usr/local/lib/libturbojpeg.so'
