--- a/getmac/__main__.py
+++ b/getmac/__main__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/opt/bin/python
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function
--- a/getmac/getmac.py
+++ b/getmac/getmac.py
@@ -131,7 +131,7 @@ FORCE_METHOD = ""  # type: str
 # The process environment does not change after a process is started
 PATH = os.environ.get("PATH", os.defpath).split(os.pathsep)  # type: List[str]
 if not WINDOWS:
-    PATH.extend(("/sbin", "/usr/sbin"))
+    PATH.extend(("/opt/sbin", "/opt/usr/sbin", "/sbin", "/usr/sbin"))
 else:
     # TODO: Prevent edge case on Windows where our script "getmac.exe"
     #   gets added to the path ahead of the actual Windows getmac.exe
