--- a/google_api_core.egg-info/requires.txt
+++ b/google_api_core.egg-info/requires.txt
@@ -1,4 +1,4 @@
-google-auth<2.0dev,>=1.25.0
+google-auth<3.0dev,>=1.25.0
 googleapis-common-protos<2.0dev,>=1.6.0
 packaging>=14.3
 protobuf>=3.12.0
--- a/setup.py
+++ b/setup.py
@@ -31,7 +31,7 @@ release_status = "Development Status ::
 dependencies = [
     "googleapis-common-protos >= 1.6.0, < 2.0dev",
     "protobuf >= 3.12.0",
-    "google-auth >= 1.25.0, < 2.0dev",
+    "google-auth >= 1.25.0, < 3.0dev",
     "requests >= 2.18.0, < 3.0.0dev",
     "setuptools >= 40.3.0",
     "packaging >= 14.3",
