--- a/pandas/core/dtypes/dtypes.py
+++ b/pandas/core/dtypes/dtypes.py
@@ -662,8 +662,8 @@ class DatetimeTZDtype(PandasExtensionDty
     >>> pd.DatetimeTZDtype(tz='UTC')
     datetime64[ns, UTC]
 
-    >>> pd.DatetimeTZDtype(tz='dateutil/US/Central')
-    datetime64[ns, tzfile('/usr/share/zoneinfo/US/Central')]
+    >>> pd.DatetimeTZDtype(tz='dateutil/Europe/London')
+    datetime64[ns, tzfile('/opt/share/zoneinfo/Europe/London')]
     """
 
     type: type[Timestamp] = Timestamp
