--- a/meson.build
+++ b/meson.build
@@ -2,7 +2,7 @@
 project(
     'pandas',
     'c', 'cpp', 'cython',
-    version: run_command(['generate_version.py', '--print'], check: true).stdout().strip(),
+    version: '@VERSION@',
     license: 'BSD-3',
     meson_version: '>=1.2.1',
     default_options: [
--- a/pandas/meson.build
+++ b/pandas/meson.build
@@ -1,23 +1,5 @@
-incdir_numpy = run_command(py,
-  [
-    '-c',
-    '''
-import os
-import numpy as np
-try:
-    # Check if include directory is inside the pandas dir
-    # e.g. a venv created inside the pandas dir
-    # If so, convert it to a relative path
-    incdir = os.path.relpath(np.get_include())
-except Exception:
-    incdir = np.get_include()
-print(incdir)
-     '''
-  ],
-  check: true
-).stdout().strip()
 
-inc_np = include_directories(incdir_numpy)
+inc_np = include_directories('@STAGING_DIR@/opt/include')
 inc_pd = include_directories('_libs/include')
 
 fs.copyfile('__init__.py')
