--- a/hass_nabucasa.egg-info/requires.txt
+++ b/hass_nabucasa.egg-info/requires.txt
@@ -1,6 +1,6 @@
 pycognito==0.1.4
 snitun==0.20
-acme==1.9.0
+acme>=1.9.0
 cryptography<4.0,>=2.8
 attrs>=19.3
 pytz>=2019.3
--- a/setup.py
+++ b/setup.py
@@ -30,7 +30,7 @@ setup(
     install_requires=[
         "pycognito==0.1.4",
         "snitun==0.20",
-        "acme==1.9.0",
+        "acme>=1.9.0",
         "cryptography>=2.8,<4.0",
         "attrs>=19.3",
         "pytz>=2019.3",
