--- a/homeassistant/scripts/__init__.py
+++ b/homeassistant/scripts/__init__.py
@@ -28,6 +28,8 @@ def run(args: list[str]) -> int:
 
         if os.path.isdir(os.path.join(path, fil)):
             scripts.append(fil)
+        elif fil != "__init__.pyc" and fil.endswith(".pyc"):
+            scripts.append(fil[:-4])
         elif fil != "__init__.py" and fil.endswith(".py"):
             scripts.append(fil[:-3])
 
