--- a/homeassistant/package_constraints.txt
+++ b/homeassistant/package_constraints.txt
@@ -34,7 +34,7 @@ voluptuous==0.12.2
 yarl==1.6.3
 zeroconf==0.36.8
 
-pycryptodome>=3.6.6
+pycryptodome==3.9.7
 
 # Constrain urllib3 to ensure we deal with CVE-2019-11236 & CVE-2019-11324
 urllib3>=1.24.3
--- a/script/gen_requirements_all.py
+++ b/script/gen_requirements_all.py
@@ -61,7 +61,7 @@ CONSTRAINT_PATH = os.path.join(
     os.path.dirname(__file__), "../homeassistant/package_constraints.txt"
 )
 CONSTRAINT_BASE = """
-pycryptodome>=3.6.6
+pycryptodome==3.9.7
 
 # Constrain urllib3 to ensure we deal with CVE-2019-11236 & CVE-2019-11324
 urllib3>=1.24.3
