--- a/homeassistant/package_constraints.txt
+++ b/homeassistant/package_constraints.txt
@@ -6,7 +6,7 @@ aiohttp-fast-url-dispatcher==0.3.0
 aiohttp-zlib-ng==0.3.1
 aiohttp==3.9.3
 aiohttp_cors==0.7.0
-astral==2.2
+astral>=2.2,<3.3
 async-interrupt==1.1.1
 async-upnp-client==0.38.2
 atomicwrites-homeassistant==1.4.1
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -27,7 +27,7 @@ dependencies    = [
     "aiohttp_cors==0.7.0",
     "aiohttp-fast-url-dispatcher==0.3.0",
     "aiohttp-zlib-ng==0.3.1",
-    "astral==2.2",
+    "astral>=2.2,<3.3",
     "async-interrupt==1.1.1",
     "attrs==23.2.0",
     "atomicwrites-homeassistant==1.4.1",
