--- a/homeassistant/package_constraints.txt
+++ b/homeassistant/package_constraints.txt
@@ -1,7 +1,7 @@
 aiodiscover==1.5.1
 aiohttp==3.8.5
 aiohttp_cors==0.7.0
-astral==2.2
+astral>=2.2,<3.3
 async-upnp-client==0.36.2
 atomicwrites-homeassistant==1.4.1
 attrs==23.1.0
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,7 @@ classifiers = [
 requires-python = ">=3.11.0"
 dependencies    = [
     "aiohttp==3.8.5",
-    "astral==2.2",
+    "astral>=2.2,<3.3",
     "attrs==23.1.0",
     "atomicwrites-homeassistant==1.4.1",
     "awesomeversion==23.8.0",
