--- a/setup.cfg
+++ b/setup.cfg
@@ -1,8 +1,35 @@
 [metadata]
+name = homeassistant
 version = 2022.6.5
+author = The Home Assistant Authors
+author_email = hello@home-assistant.io
+license = Apache-2.0
+platforms = any
+description = Open-source home automation platform running on Python 3.
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+keywords = home, automation
 url = https://www.home-assistant.io/
+project_urls = 
+	Source Code = https://github.com/home-assistant/core
+	Bug Reports = https://github.com/home-assistant/core/issues
+	Docs: Dev   = https://developers.home-assistant.io/
+	Discord     = https://www.home-assistant.io/join-chat/
+	Forum       = https://community.home-assistant.io/
+classifier = 
+	Development Status :: 5 - Production/Stable
+	Intended Audience :: End Users/Desktop
+	Intended Audience :: Developers
+	License :: OSI Approved :: Apache Software License
+	Operating System :: OS Independent
+	Programming Language :: Python :: 3.9
+	Programming Language :: Python :: 3.10
+	Topic :: Home Automation
 
 [options]
+packages = find:
+zip_safe = False
+include_package_data = True
 python_requires = >=3.9.0
 install_requires = 
 	aiohttp==3.8.1
@@ -28,6 +55,14 @@ install_requires =
 	voluptuous-serialize==2.5.0
 	yarl==1.7.2
 
+[options.packages.find]
+include = 
+	homeassistant*
+
+[options.entry_points]
+console_scripts = 
+	hass = homeassistant.__main__:main
+
 [flake8]
 exclude = .venv,.git,.tox,docs,venv,bin,lib,deps,build
 max-complexity = 25
--- /dev/null
+++ b/setup.py
@@ -0,0 +1,2 @@
+from setuptools import setup
+setup()
