--- a/homeassistant/util/package.py
+++ b/homeassistant/util/package.py
@@ -92,7 +92,7 @@ def install_package(
         env["PYTHONUSERBASE"] = os.path.abspath(target)
         # Workaround for incompatible prefix setting
         # See http://stackoverflow.com/a/4495175
-        args += ["--prefix="]
+        # args += ["--prefix="]
     _LOGGER.debug("Running pip command: args=%s", args)
     with Popen(args, stdin=PIPE, stdout=PIPE, stderr=PIPE, env=env) as process:
         _, stderr = process.communicate()
