--- /dev/null
+++ b/setup.py
@@ -0,0 +1,58 @@
+
+from setuptools import find_packages, setup
+
+version = '1.3.0'
+
+install_requires = [
+    'fe25519~=1.3',
+]
+
+coveralls_extras = [
+    'coveralls~=3.3.1',
+]
+
+docs_extras = [
+    'toml~=0.10.2',
+    'sphinx~=4.2.0',
+    'sphinx-rtd-theme~=1.0.0',
+]
+
+lint_extras = [
+    'pylint~=2.14.0',
+]
+
+publish_extras = [
+    'build~=0.8',
+    'twine~=4.0',
+]
+
+test_requires = [
+    'pytest~=7.0',
+    'pytest-cov~=3.0',
+    'parts~=1.5',
+    'bitlist~=1.0',
+    'fountains~=2.0',
+]
+
+setup(
+    name='ge25519',
+    version=version,
+    description='Pure-Python data structure for working with Ed25519 (and Ristretto) group elements and operations.',
+    url='https://github.com/nthparty/ge25519',
+    author='Andrei Lapets',
+    author_email='a@lapets.io',
+    license='MIT',
+    python_requires='>=3.7',
+
+    packages=find_packages(where='src'),
+    package_dir={'': 'src'},
+    include_package_data=True,
+    install_requires=install_requires,
+    extras_require={
+        'coveralls': coveralls_extras,
+        'docs': docs_extras,
+        'lint': lint_extras,
+        'publish': publish_extras,
+        'test': test_requires,
+    },
+)
