--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,7 +5,7 @@ build-backend = "hatchling.build"
 [project]
 name = "httpx"
 description = "The next generation HTTP client."
-license = "BSD-3-Clause"
+license = {file = "LICENSE.md"}
 requires-python = ">=3.7"
 authors = [
     { name = "Tom Christie", email = "tom@tomchristie.com" },
@@ -33,7 +33,7 @@ dependencies = [
     "idna",
     "sniffio",
 ]
-dynamic = ["readme", "version"]
+dynamic = ["version"]
 
 [project.optional-dependencies]
 brotli = [
