--- a/setup.py
+++ b/setup.py
@@ -19,6 +19,7 @@ class prepare_tinydtls(build_ext):
         commands =  [
                     ["sh", "-c", "autoconf"],
                     ["sh", "-c", "autoheader"],
+                    ["sed", "-i", "-e", "s,cross_compiling=no,cross_compiling=yes,", "configure"], # CROSS
                     ["sh", "-c", "./configure"], # no --without-ecc
                     ]
         if not os.path.exists(os.path.join(os.path.dirname(__file__), 'DTLSSocket','tinydtls','dtls.c')):
